src/main/java/controller/GameManager.java

```diff
@@ -0,0 +1,79 @@
+package controller;
+
+import controller.dto.JudgeResult;
+import controller.dto.ParticipantHandStatus;
+import domain.constants.CardCommand;
+import domain.game.ActionAfterPick;
+import domain.game.BlackJackGame;
+import domain.game.DecisionToContinue;
+import domain.participant.Participant;
+import domain.participant.Player;
+import java.util.List;
+import view.InputView;
+import view.OutputView;
+
+public class GameManager {
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public GameManager(final InputView inputView, final OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        BlackJackGame blackJackGame = start();
+        rotate(blackJackGame);
+        finish(blackJackGame);
+    }
+
+    /**
+     * 게임의 참가자를 생성하고 2장 씩 나누어주어 게임을 초기 세팅하는 메서드
+     */
+    private BlackJackGame start() {
+        BlackJackGame blackJackGame = BlackJackGame.from(inputView.enterPlayerNames());
+        outputView.printInitialHandStatus(blackJackGame.initialize());
+        return blackJackGame;
+    }
+
+    /**
+     * 딜러와 플레이어에게 돌아가며 카드를 나누어주는 메서드
+     */
+    private void rotate(final BlackJackGame blackJackGame) {
+        List<Participant> participants = blackJackGame.getParticipants();
+        for (Participant participant : participants) {
+            blackJackGame.giveCard(
+                    participant,
+                    getAction(participant),
+                    getSupplier(participant)
```

Comment:
Functional Interface를 사용하면서 어떤 행동을 하는지 의미가 퇴색되었다고 생각합니다.
의미있는 이름을 부여할 방법은 없을까요? 🤔

Reply:
[반영 완료] 

해당 부분에서 메서드 `getAction`, `getSupplier`가 어떤 동작을 하는 지 명확하게 설명해주지 않아서 리뷰를 남겨주신 것 같아요. 아래와 같이 명확한 네이밍으로 수정하였습니다!
```suggestion
                    getActionAfterPick(participant),
                    getDecisionToContinue(participant)
```